Parse/src/main/java/com/parse/ParseHttpRequest.java

```diff
@@ -12,8 +12,15 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * The http request we send to parse server. Instances of this class are not immutable. The
+ * request body may be consumed only once. The other fields are immutable.
+ */
```

Comment:
The body of the `ParseHttpRequest` is mutable, they can be change after we build the `ParseHttpRequest`.